File: VB/WinSolution.Module/Updater.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

Namespace WinSolution.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim obj1 As DemoObject = ObjectSpace.CreateObject(Of DemoObject)()
			obj1.Name = "DemoObject1"
			Dim obj2 As DemoObject = ObjectSpace.CreateObject(Of DemoObject)()
			obj2.Name = "DemoObject2"
			Dim lookupObj1 As DemoLookupObject = ObjectSpace.CreateObject(Of DemoLookupObject)()
			lookupObj1.Name = "DemoLookupObject1"
			Dim lookupObj2 As DemoLookupObject = ObjectSpace.CreateObject(Of DemoLookupObject)()
			lookupObj2.Name = "DemoLookupObject2"
			obj1.LookupProperty = lookupObj1
			obj2.LookupProperty = lookupObj2
			obj1.Save()
			lookupObj1.Save()
			lookupObj2.Save()
		End Sub
	End Class
End Namespace

File: CS/WinSolution.Module/Updater.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace WinSolution.Module {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            DemoObject obj1 = ObjectSpace.CreateObject<DemoObject>();
            obj1.Name = "DemoObject1";
            DemoObject obj2 = ObjectSpace.CreateObject<DemoObject>();
            obj2.Name = "DemoObject2";
            DemoLookupObject lookupObj1 = ObjectSpace.CreateObject<DemoLookupObject>();
            lookupObj1.Name = "DemoLookupObject1";
            DemoLookupObject lookupObj2 = ObjectSpace.CreateObject<DemoLookupObject>();
            lookupObj2.Name = "DemoLookupObject2";
            obj1.LookupProperty = lookupObj1;
            obj2.LookupProperty = lookupObj2;
            obj1.Save();
            lookupObj1.Save();
            lookupObj2.Save();
        }
    }
}
